easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-Java-1.7.0_80.eb

```diff
@@ -17,22 +17,26 @@ description = """FastQC is a quality control application for high throughput seq
  application to review the results of several different QC checks, or create an HTML based
  report which can be integrated into a pipeline."""
 
-toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchain = {'name': 'dummy', 'version': ''}
 
 sources = ['%(namelower)s_v%(version)s.zip']
 source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
 
+patches = ['FastQC_shebang.patch']
+
 java = 'Java'
 javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
 
-dependencies = [('Java', '1.7.0_80', '', True)]
-
-install_cmd = "cp -a FastQC/* %(installdir)s && chmod +x %(installdir)s/fastqc"
+postinstallcmds = ['chmod +x %(installdir)s/fastqc']
 
 sanity_check_paths = {
     'files': ["fastqc", "fastqc_icon.ico", "INSTALL.txt", "jbzip2-0.9.jar", "LICENSE.txt",
-              "README.txt", "RELEASE_NOTES.txt", "run_fastqc.bat", "sam-1.32.jar"],
+              "README.txt", "RELEASE_NOTES.txt", "run_fastqc.bat", "sam-1.32.jar", "cisd-jhdf5.jar"],
```

Comment:
@nathanhaigh copy-paste error, the `.jar` you added isn't there yet for this FastQC version?